masonry/src/widget/image.rs

```diff
@@ -77,6 +77,8 @@ impl Widget for Image {
 
     fn on_access_event(&mut self, _ctx: &mut EventCtx, _event: &AccessEvent) {}
 
+    fn register_children(&mut self, _ctx: &mut RegisterCtx) {}
+
     fn on_status_change(&mut self, _ctx: &mut LifeCycleCtx, _event: &StatusChange) {}
 
     fn lifecycle(&mut self, _ctx: &mut LifeCycleCtx, _event: &LifeCycle) {}
```

Comment:
```suggestion
```
(and a more general question of whether this should be done for the other methods/across other widgets)

Reply:
It could be done in a separate PR. I'm going to merge lifecycle and on_status_change anyway.